src/main/resources/io/jenkins/plugins/audit/config/AuditLogConfiguration/help-pattern.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Help message is not being displayed. I am figuring out why!
```

Comment:
I have a fairly trivial plugin [here](https://github.com/jenkinsci/paranoia-plugin) which figured out how to get this help snippet thing to work.